python/ucxx/__init__.py

```diff
@@ -95,6 +95,11 @@ def _is_mig_device(handle):
     logger.info("Setting UCX_MAX_RNDV_RAILS=1")
     os.environ["UCX_MAX_RNDV_RAILS"] = "1"
 
+if "UCX_PROTO_ENABLE" not in os.environ:
+    # UCX protov2 still doesn't support CUDA async/managed memory
+    logger.info("Setting UCX_PROTO_ENABLE=n")
+    os.environ["UCX_PROTO_ENABLE"] = "n"
```

Comment:
Do we document, like ucx-py, where the UCXX settings diverge from the UCX defaults? If so, maybe do that for this one too?

Reply:
We do not have docs here yet primarily because I didn't want to maintain two copies of it, so I'm planning to copy/move up-to-date docs from UCX-Py when we're about to make the switch.